### DISCUSSION TITLE
usdc permit op eth

### DISCUSSION THREAD

**Discussion by Nithin-Varma** [community/community#108959] [Discussions]
### Select Topic Area

Bug

### Body

i want to implement permit function, its working fine when i deploy my own erc20 token, but when i use already deployed usdc on op sepolia it was giving an invalid sig error.
anyone faced this issue before?

**Comment by queenofcorgis** [comment]
Hi @Nithin-Varma ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on USDC. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
